kernel-api-occurrences.md

```diff
@@ -0,0 +1,14 @@
+
+| Kernel Directory| Kernel API | Kernel API call |Kernel API Occurrences | Get the call |
+|---|---|---|---|---|
+| Crypto | [Linux Kernel Crypto API](https://www.kernel.org/doc/html/latest/crypto/index.html) | Initialization Call  |  grep  -c 'crypto_alloc*' crypto/*| grep  -n 'crypto_alloc*' crypto/* |
+|  | | Symmetric Key Cipher API  | grep  -c 'crypto_skcipher*' crypto/* | grep  -n 'crypto_skcipher*' crypto/* |
+|  | | Single Block Cipher API | grep  -c 'crypto_cipher*' crypto/ | grep  -c 'crypto_cipher*' crypto/ |
+|  |  | Asynchronous Block Cipher API |  grep  -c 'crypto_ablkcipher*' crypto/ | grep  -n 'crypto_ablkcipher*' crypto/ |
```

Comment:
Did you find any occurrence of this in latest mainline kernel? I think this was deprecated.